pkg/hubble/parser/options/options.go

```diff
@@ -60,6 +61,18 @@ func Redact(logger logrus.FieldLogger, httpQuery, httpUserInfo, kafkaApiKey bool
 	}
 }
 
+// EnableL3L4 configures the Network Policy enrichment.
+func EnableL3L4(logger logrus.FieldLogger, enableL3L4 bool) Option {
```

Comment:
Rename function to `EnableL3L4PolicyEnrichment`.

Reply:
Done